docs/source/authorization.rst

```diff
@@ -0,0 +1,131 @@
+.. _authorization:
+
+Authorization
```

Comment:
I think it's authentication, not authorization. In every place :D

Reply:
It's actually often used in other libraries (some libs use authentication and some authorization, depending on the context..). It's safer to go with authentication then.